editor/docks/scene_tree_dock.cpp

```diff
@@ -2911,6 +2923,18 @@ void SceneTreeDock::_selection_changed() {
 		_push_item(nullptr);
 	}
 
+	// Track script changes in selected nodes
```

Comment:
```suggestion
	// Track script changes in selected nodes.
```
Also this comment should be when connecting the signal I think.

Reply:
Done.